IIPS/iip-voting_strat_change.md

```diff
@@ -0,0 +1,57 @@
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+Include unclaimed INDEX rewards and INDEX in LP pools in a user’s vote weight in snapshot governance. This allows both small holders who cannot often claim due to gas costs to have the voting power they deserve and large holders to increase the liquidity on INDEX pairs without reducing their governance power.
+
+## Abstract
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the IIP is implemented, not *why* it should be done or *how* it will be done. If the IIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+Index Coop’s current Snapshot vote-counting strategy simply weights your votes based on how many INDEX tokens an address owns. Many users have unclaimed tokens from the UNI LP farming program and tokens locked in INDEX based LP pools. These votes are not eligible to vote on Snapshot governance. This proposal would allow unclaimed tokens and tokens in LP positions to be included in Snapshot votes.
+
+
+## Motivation
+<!--This is the problem statement. This is the *why* of the IIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the IIP proposes changing how something is calculated, you must address *why* the current calculation is inaccurate or wrong. This is not the place to describe how the IIP will address the issue!-->
+As of now, unclaimed rewards not being counted in the vote is a very minor problem, but as the emission rate of INDEX rewards slows, it becomes more apparent. For example, if in the future, farming programs may produce a more modest 10% APY. A user staking just $500 worth of DPI would make $50 in INDEX rewards each year. At the time of writing this, claiming your rewards costs ~$5. This would make it highly inefficient to claim more than once or twice per year, significantly diluting the voting power of this user between claims.
+
+Many large holder may lock their INDEX in liquidity pools. This benefits the Index ecosystem by reducing slippage in INDEX trades. Unfortunately, these holders cannot participate in governance, even though they may be significantly invested in INDEX.
+
+#### For:
+- Gives small holders their fair vote
+- Increases liquidity in INDEX trading pairs
+
+#### Against:
+- Adds additional complexity to our Snapshot governance
+- Only affects a small proportion of DPI/INDEX holders
```

Comment:
The for and against sections should specify exactly what changes to the system are being voted on when a user votes *for* or *against*.
Example: https://github.com/SetProtocol/IIPS/pull/11/files#diff-64553660f36c608c65a9f7f0063a081908be7552c842a9fad062cb1ff885baa3R18-R24